src/Controller/InitializeController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Controller;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Handles the Contao initialize.php route.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class InitializeController extends Controller
+{
+    /**
+     * Runs the initialize action for legacy entry point scripts.
+     *
+     * @return Response
+     *
+     * @Route("/_initialize", name="contao_initialize")
+     */
+    public function indexAction()
+    {
+        trigger_error('Custom entry points are deprecated and will no longer work in Contao 5.0.', E_USER_DEPRECATED);
+
+        $masterRequest = $this->get('request_stack')->getMasterRequest();
+        $realRequest   = Request::createFromGlobals();
+        $scope         = ContaoCoreBundle::SCOPE_FRONTEND;
+
+        if (defined('TL_MODE') && TL_MODE === 'BE') {
+            $scope = ContaoCoreBundle::SCOPE_BACKEND;
+        }
+
+        // Necessary to make the base path correct
```

Comment:
The method `Request::getBasePath()` would return the path to the entry point script as relative URL all assets.